xsgen/openmc_origen.py

```diff
@@ -559,19 +559,19 @@ def _parse_statepoint(self, statepoint, tally_id=1):
         e_g : list of floats
             Group structure.
         """
-        sp = StatePoint(statepoint)
-        sp.read_results()
+        sp = statepoint.StatePoint(statepoint_path)
+        print(sp.tallies)
         # compute group fluxes for data sources
-        for tally, ds in zip(sp.tallies[1:3], (self.eafds, self.omcds)):
-            ds.src_phi_g = tally.results[::-1, :, 0].flatten()
-            ds.src_phi_g /= ds.src_phi_g.sum()
+        #for tally, ds in zip(sp.tallies[1:3], (self.eafds, self.omcds)):
+            #ds.src_phi_g = tally.results[::-1, :, 0].flatten()
+            #ds.src_phi_g /= ds.src_phi_g.sum()
```

Comment:
On the first iteration through the loop it is the eafds and on the second iteration it the omcds.  See Python's [zip() fucntion](https://docs.python.org/3/library/functions.html#zip)